src/router.js

```diff
@@ -47,22 +47,25 @@ function match(pathname, routes) {
   for (var i = 0; i < routes.length && !match; i++) {
     var route = routes[i][0]
     var keys = []
-
     pathname.replace(
       RegExp(
         route === "*"
           ? ".*"
           : "^" +
             route.replace(/\//g, "\\/").replace(/:([\w]+)/g, function(_, key) {
               keys.push(key)
-              return "([-\\.%\\w]+)"
+              return "([-\\.%\\w\\(\\)]+)"
```

Comment:
it is the necessary escaping for the open and close parens.
`\\` gets converted to a single `\` and then when the regexp is run that single `\` escapes the `(` and `)`.  (I followed the syntax for the `.`)